README.md

```diff
@@ -1,12 +1,40 @@
-<h2 align="center"><img src="images/ukat_logo.png" height="180"></h2>
+<h2 align="center"><img src="https://raw.githubusercontent.com/UKRIN-MAPS/ukat/master/images/ukat_logo.png" height="180"></h2>
 <p align="center"><strong>UKRIN Kidney Analysis Toolbox (ukat) </strong></p>
 
 ![Build and Test](https://github.com/UKRIN-MAPS/ukat/workflows/Build%20and%20Test/badge.svg?branch=master)
 [![codecov](https://codecov.io/gh/UKRIN-MAPS/ukat/branch/master/graph/badge.svg?token=QJ9DQONJBP)](https://codecov.io/gh/UKRIN-MAPS/ukat)
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 [![DOI](https://zenodo.org/badge/256993671.svg)](https://zenodo.org/badge/latestdoi/256993671)
 
-# Getting started
+`ukat` is a vendor agnostic framework for the analysis of quantitative renal MRI data. 
+
+The [UKRIN-MAPS project](https://www.nottingham.ac.uk/research/groups/spmic/research/uk-renal-imaging-network/ukrin-maps.aspx) aims to standardise the acquisition and analysis of renal MRI data to enable multi-site, 
+multi-vendor studies. 
+Although many MRI vendors produce quantitative maps on the scanner, their methods are closed source and as such, 
+potentially cause variability in multi-vendor studies. `ukat` provides an open-source and robust analysis platform that can be used to process data from multiple vendors. 
+
+The focus of this package is analysis of data from the standardised UKRIN protocol, however the methods are intentionally 
+left generic to enable analysis of data collected using different protocols or on different areas of the body.
+
+More information can be found in [this ISMRM abstract](https://www.researchgate.net/publication/349830229_UKRIN_Kidney_Analysis_Toolbox_UKAT_A_Framework_for_Harmonized_Quantitative_Renal_MRI_Analysis).
+
+# Installing `ukat`
+There are a few different ways you can install `uakt` based on what you want to do with it
+### "I just want to process my data with this package"
+1. Make sure you're running Python >=3.6
+2. Install `ukat` with `pip install ukat`
+
+### "I want to modify this code to do something a bit different but don't want my modifications to go back into `ukat`"
+1. Clone this reposetry with `git clone https://github.com/UKRIN-MAPS/ukat.git`
+2. Change to the `ukat` root directory (the one containing a file named `setup.py`).
+3. Run the following command in your terminal: `pip install -e . `
+4. Install [SimpleElastix](https://simpleelastix.github.io/) ([instructions](https://simpleelastix.readthedocs.io/GettingStarted.html)).
```

Comment:
I would remove this SimpleElastix part for now. We have no motion correction algorithms at the moment in ukat and we might use ITK-Elastix package or no elastix python package at all in the future.